test/test.mixins.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+const describe = require('mocha').describe;
+const it = require('mocha').it;
+const expect = require('chai').expect;
+const path = require('path');
+
+// global config object for each test
+const conf = require('confi')({
+  path: [
+    './conf'
+  ],
+  context: {
+    CKDIR: __dirname,
+    CONFIGDIR: path.join(process.cwd(), 'clientkit')
+  }
+});
+conf.consoleOnly = true;
+
+describe('bg colors mixin', function() {
+  it('generates css bg colors', (done) => {
+    const colors = require('../styles/mixins/bg-colors.js')(conf);
+    const result = colors({});
+    expect(result['.bg-default']['background-color']).to.equal(conf.color['background-default']);
+    done();
+  });
+});
+
+describe('bg image mixin', function() {
+  it('generates css bg image', (done) => {
+    const image = require('../styles/mixins/bg-image.js')(conf);
+    const result = image({}, 'http://nope.com/myImage.jpg');
+    expect(result['background-image']).to.equal('url(http://nope.com/myImage.jpg)');
+    done();
+  });
+});
+
+describe('buttons mixin', function() {
+  this.timeout(20000);
+  it('generates css for buttons', (done) => {
+    const button = require('../styles/mixins/button.js')(conf);
+    // mixin, bgColor, fgColor, type
+    const result = button({}, '#ff0000', '#00ff00', 'outline');
+    expect(result.color).to.equal('#ff0000');
+    expect(result.border).to.equal('2px solid #ff0000');
+    expect(result['&:hover'].color).to.equal('#00ff00');
+    done();
+  });
+});
+describe('buttons outline mixin', function() {
+  this.timeout(20000);
+  it('generates css outlines for buttons', (done) => {
+    const button = require('../styles/mixins/button-outline.js')(conf);
```

Comment:
make sure you are looking at master, I don't think button-outline exists anymore